src/components/block-tunes/block-tune-delete.ts

```diff
@@ -63,23 +64,15 @@ export default class DeleteTune implements BlockTune {
   }
 
   /**
-   * Create "Delete" button and add click event listener
-   *
-   * @returns {HTMLElement}
+   * Tune's appearance in block settings menu
    */
-  public render(): HTMLElement {
-    this.nodes.button = $.make('div', [this.CSS.button, this.CSS.buttonDelete], {});
-    this.nodes.button.appendChild($.svg('cross', 12, 12));
-    this.api.listeners.on(this.nodes.button, 'click', (event: MouseEvent) => this.handleClick(event), false);
-
-    /**
-     * Enable tooltip module
-     */
-    this.api.tooltip.onHover(this.nodes.button, this.api.i18n.t('Delete'), {
-      hidingDelay: 300,
-    });
-
-    return this.nodes.button;
+  public render(): PopoverItem {
```

Comment:
I changed my mind regarding the need of separate getter for popover items. I decided that tool should not have 2 ways of configuring its appearance in tunes menu at the same time. 
I happened because i could not resolve the question how to handle situation when tool has `renderSettings()` returning html and `toolbox()` returning array of popover items at once